### PR TITLE
cant stand (disabled) (prism disabling reference)

### PR DESCRIPTION
**Pull request by NatalieCrazy**
uhhhhh if this is absolute garbage im sorry

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2057,15 +2072,69 @@ def __init__(self):
...
+            if self.prismname not in owned_prisms_name:
                 await interaction.response.send_message("you dont even have that prism what", ephemeral=True)
                 return
-            # oh god i have to make a thing actually do a thing now please i ask for forgiveness
+            # i ask for forgiveness
             embedVar = discord.Embed(title=f"Configure {self.prismname}", description="Turn off any boosts from your prism that you don't want", color=0x6E593C)
-                
+            count = 1
+            for prism in owned_prisms:
+                if prism.name = self.prismname:
```
Comment (ZakChrom) [inline main.py:2083]: ==

**Review by ZakChrom — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (NatalieCrazy) [reply main.py:2083]: its joever

**Comment by NatalieCrazy** [comment]
ok i think we good now

**Review by milenakos — APPROVED** [review]
istg if this will still crash

**Comment by MonkeysWithPie** [comment]
good luck

**Pull request merged**